internal/compliance/snapshot_poller/pollers/aws/configservice.go

```diff
@@ -150,6 +155,14 @@ func buildConfigServiceSnapshot(
 		RecordingGroup: recorder.RecordingGroup,
 		RoleARN:        recorder.RoleARN,
 	}
+	// Check if ResourceID matches the integration's regex filter
+	if pollerInput != nil {
+		if ignore, err := pollerInput.ShouldIgnoreResource(*recorder.Name); ignore || err != nil {
+			if err != nil || ignore {
+				return nil, err
+			}
+		}
```

Comment:
These nested `if` statements are now redundant (I'm assuming this is refactoring leftovers)
```suggestion
		if ignore, err := pollerInput.ShouldIgnoreResource(*recorder.Name); ignore || err != nil {
			return nil, err
		}
```